app/forms/draft_work_form.rb

```diff
@@ -61,6 +62,12 @@ def deserialize_embargo(params)
   end
   # rubocop:enable Metrics/AbcSize
 
+  def access_from_collection(params)
+    return if model.collection.access == 'depositor-selects'
+
+    params['access'] = model.collection.access
```

Comment:
Can you add a request spec that tests this case?

Reply:
Let me know if the additions to the reqeust/create_new_work spec are not sufficient.

Reply:
All of the test I see are doing the "depositor-selects" branch of this method.  I don't see a test for "the user submitted something that isn't allowed and we overwrote it".